Scoop/View/EditProfileViewController.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  EditProfileViewController.swift
+//  Scoop
+//
+//  Created by Caitlyn Jin on 9/27/23.
+//
+
+import SDWebImage
+import UIKit
+
+class EditProfileViewController: UIViewController {
+    
+    // MARK: - Views
+    
+    private let profileImageView = UIImageView()
+    private let uploadPhotoButton = UIButton()
+    
+    private let stackView = UIStackView()
+    private let nameTextField = UITextField()
+    private let pronounsTextField = UITextField()
+    private let hometownTextField = UITextField()
+    private let classTextField = UITextField()
+    
+    weak var delegate: ProfileViewDelegate?
+    
+    // MARK: - User Data
+    
+    private var user: BaseUser?
+    
+    private var hometown: String?
+    private var talkative: Float?
+    private var music: Float?
+    private var snack: String?
+    private var song: String?
+    private var stop: String?
+    
+    // MARK: - Lifecycle Functions
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.white
```

Comment:
Try to keep the color usage consistent. I usually just go with ".white" except when I'm setting a border, but it's alright as long as you keep it consistent throughout the file. I think later on you used ".red"